### ISSUE TITLE
Toogle button accessibility: the state is not read by screen readers.

### ISSUE THREAD

**Issue by DavidBailes** [audacity/audacity#11630]
### Bug description

This issue affects all toggle buttons, eg the loop region toggle button in the toolbar, the power/bypass toggle buttons in the real time effect panel.

### Steps to reproduce

With a screen reader running:
1. Using keyboard navigation move to the loop region toggle button.
2. Press Enter.

### Expected behavior

The state of the toggle button should be read both when the button gains focus, and when its state is changed by pressing Enter. (eg on/off, or pressed/not pressed, or checked/not checked)

### Actual behavior

The state is not read.

### Audacity Version

Other

### Operating system

Windows

### Additional context

Tested with commit: 1de3894.

This issue also occurs on Linux, but I haven't tested on macOS.

On Windows, this is a regression wrt Audacity 3.
